common/src/main/java/org/conscrypt/OpenSSLCipher.java

```diff
@@ -1196,7 +1196,10 @@ protected AlgorithmParameters engineGetParameters() {
                 AlgorithmParameters params = AlgorithmParameters.getInstance("GCM");
                 params.init(spec);
                 return params;
-            } catch (NoSuchAlgorithmException | InvalidParameterSpecException e) {
+            } catch (NoSuchAlgorithmException e) {
+                // This may happen since Conscrypt doesn't provide this itself.
```

Comment:
Yeah, just updating the comment is all that's necessary.  Optionally, you could throw AssertionError() to enforce that we always provide it, but it's probably not necessary.